scipy/spatial/distance.py

```diff
@@ -888,8 +886,12 @@ def seuclidean(u, v, V):
     """
     Return the standardized Euclidean distance between two 1-D arrays.
 
-    The standardized Euclidean distance between `u` and `v`.
+    Computes the standardized Euclidean distance. The standardized Euclidean distance between two n-vectors u and v is
```

Comment:
Besides the line length, input parameters need to have single back ticks.
```suggestion
    Computes the standardized Euclidean distance. The standardized
    Euclidean distance between two n-vectors `u` and `v` is
```

Reply:
Sorry for not being clear, it has to go in the commit message.

e.g. if you accept all the suggestions I made, you can go in the `Files changed` tab, batch all suggestions and when clicking on the commit button you will be able to change the message. Here write a proper message and you can add the skip marks.